lib/rubocop/cop/sorbet/signatures/enforce_signatures.rb

```diff
@@ -76,6 +78,17 @@ def check_node(node)
           @last_sig_for_scope[scope] = nil
         end
 
+        def has_rbs_comment?(node)
+          node = node.parent while RuboCop::AST::SendNode === node.parent
```

Comment:
For the case of visibility / methods such as: 
```ruby 
private def some_method; end
foo bar baz def some_method; end
```
Is there a better way of doing this?

Reply:
Would it make it easier to put the comment into `@last_sig_for_scope[scope]` as we visit the parent nodes?

Reply:
Okay, let's keep the recursion on receivers 👍